frontend/src/pages/Traces/TracesPage.tsx

```diff
@@ -108,7 +108,7 @@ export const TracesPage: React.FC = () => {
 		startDate: searchTimeContext.startDate,
 		endDate: searchTimeContext.endDate,
 		skipPolling,
-		sortColumn,
+		sortColumn: sortColumn || undefined,
```

Comment:
Nit, but isn't this equivalent to `sortColumn,`?

Reply:
Technically it can be `null` so this was just to be type happy, but open to other suggestions